Documentation/k8s-pre-reqs.md

```diff
@@ -6,6 +6,8 @@ indent: true
 
 # Prerequisites
 
+Rook can be installed on any existing Kubernetes clusters as long as it meets the minimum version and have the required priviledge to run in the cluster (see below for more information). If you dont have a Kubernetes cluster, you can quickly set one up using [Minikube](#minikube), [Kubeadm](#kubeadm) or our core-os [vagrant](#new-local-kubernetes-cluster-with-vagrant)
```

Comment:
I'd rephrase this as:
```
using [Minikube](#minikube), [Kubeadm](#kubeadm) or [CoreOS/Vagrant](#new-local-kubernetes-cluster-with-vagrant).
```

This sentence should also end with a period.

Reply:
done